build/charts/antrea/values.yaml

```diff
@@ -67,6 +67,28 @@ ovs:
   # configuration).
   hwOffload: false
 
+secondaryNetworkOVS:
+  # Name of the secondary network OVS bridge1 antrea-agent's podWatch controller will create and use.
+  bridgeName1: "br-int1"
+  # Name of the secondary network OVS bridge2 antrea-agent's podWatch controller will create and use.
+  bridgeName2: "br-int2"
+  # Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
+    # - system
+    # - netdev
+  # 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
+  # OVS in userspace mode. Userspace mode requires the tun device driver to be available.
+  ovsDatapathType: "system"
+  # Name of the OVS patch port which would connected two OVS bridges per secondary interface config.
+  ovsPatchPort: "patch0"
```

Comment:
If the port will result in an interface created and the name cannot conflict with other ports/interfaces, probably add some prefix to make sure it is unique, like "br-secnet-pactch0"

Reply:
Added "br-secnet-" as a prefix.